cmd/kubelet/app/options/options.go

```diff
@@ -79,6 +82,8 @@ func NewKubeletServer() *KubeletServer {
 
 // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&s.Standalone, "standalone", s.Standalone, "Runs the Kubelet in 'standalone mode', e.g. without an API server. [default=false]")
```

Comment:
Could we have a little more descriptive description here? It's obvious that --standalone triggers standalone mode, but explain what it's used for and what effects it has please